src/file_change_handler.ts

```diff
@@ -71,4 +80,12 @@ export class FileChangeHandler {
 			id: "" // TODO: Fix this, should be optional!
 		}
 	}
+
+	private isDartLike(document: vscode.TextDocument): boolean {
```

Comment:
I posted [a question about this on the analyzer-discuss list](https://groups.google.com/a/dartlang.org/forum/#!topic/analyzer-discuss/AOPDiiBjxKI) and Brian suggested a couple more file types are monitored. I was sending them all for now thinking the service would just drop those it didn't care about (other than with the multi-cursor hack the traffic is minimal).

If you think it's worth doing, I think we should include all the files Brian mentioned [here](https://groups.google.com/a/dartlang.org/d/msg/analyzer-discuss/AOPDiiBjxKI/XM7naJ8KBwAJ).